esx_service/vmdk_ops_test.py

```diff
@@ -1450,6 +1429,94 @@ def testPolicyUsageCount(self):
 
         vsan_policy.delete(self.name)
 
+class VMListenerTest(unittest.TestCase):
+    """ Unit test for VM listener. """
+    logging.info("Running VMListenerTest")
+    default_tenant_vol1_name = "stale_volume"
+
+    vm1_name = generate_test_vm_name()
+    vm1 = None
+    vm1_config_path = None
+
+    datastore_name = None
+    datastore_path = None
+
+    def setUp(self):
+        """ Setup run before each test """
+        logging.info("VMListenerTest setUp path =%s", path)
+
+        if not self.datastore_name:
+            datastores = vmdk_utils.get_datastores()
+            datastore = datastores[0]
```

Comment:
the logic here seems wrong. 
I'd expect "for each DB, check if dockvol is there or can be created. if not continue. If yes, use it" 

I guess there are some assumptions about test setup here (or copy-paste from other places with assumptions :-)
If that's the case, it's worth at least commenting